### ISSUE TITLE
Running on embedded devices?

### ISSUE THREAD

**Issue by david-res** [breakfastquay/rubberband#110]
Would this library be able to run on an embedded MCU such as an STM32F7 or an NXP iMXRT1060?
I'm looking to do live time stretching of audio with key lock from a wav file playing off an SD card.

I can currently play audio and change the tempo, but can't lock the pitch

**Comment by cannam** [comment]
I don't think so. As I recall, it was (just) possible to get it running a stereo stream in real-time on the iPhone 4 / Galaxy S era of phones, being roughly 1GHz Cortex A8 including NEON instructions. I think that is significantly more powerful than what you're describing?

**Comment by david-res** [comment]
Yeah, the chips I mentioned use M7 Cortex cores running at up to 600Mhz, so if it barely cut's it on an A8, then these are defiantly not up to the task
Thanks!
